bin/pgvectors-docker-image-ecr-deployment-cdk.ts

```diff
@@ -25,6 +25,7 @@ for (const cdkRegion of cdkRegions) {
             },
             repositoryName: `${process.env.ECR_REPOSITORY_NAME}-${environment}` || 'pgvectors-docker-image-ecr-deployment-cdk',
             appName: process.env.APP_NAME || 'pgvectors',
+            imageVersion: process.env.IMAGE_VERSION || 'latest',
```

Comment:
**Suggestion:** Use nullish coalescing operator instead of logical OR for default values to handle empty strings appropriately.
```suggestion
            repositoryName: `${process.env.ECR_REPOSITORY_NAME}-${environment}` ?? 'pgvectors-docker-image-ecr-deployment-cdk',
            appName: process.env.APP_NAME ?? 'pgvectors',
            imageVersion: process.env.IMAGE_VERSION ?? 'latest',
```